21IanuarieHW/src/StoreProblem.java

```diff
@@ -0,0 +1,83 @@
+import java.util.Scanner;
+import java.util.ArrayList;
+import java.util.List;
+
+class Product {
```

Comment:
you have the right idea:
Product parent class with common attributes (don't forget id and maybe quantity). Create constructor and getters/setters
AnimlaProduct and VeggieProduct children classes with custom attributes (and constructor and getters/setters)

To make display-ing simpler consider overriding the toString method.

You will need an Order class to keep track of the sales you make.

You will need a Store class that will have a List of products and orders and it will expose methods to add/sell/list the products or orders. It should validate if an order is valid and process it (decrease the quantity of product)

You will need another class with a main function that will create a new instance of the Store and then display the menu, interract with the console and call the relevant functions from the store.

Each class should be public and in a separate file.